README.md

```diff
@@ -10,7 +34,7 @@ import abacatepay
 token = "<your enviroment api token>"
 client = AbacatePay(token)
 
-billing = client.CreateBilling(products=[Product(externalId="123", name="Teste", quantity=1, price=101, description="Teste")], returnURL="https://abacatepay.com", completionUrl="https://abacatepay.com")
```

Comment:
I think we could remove this `x =`, since it's unnecessary

See https://github.com/AbacatePay/abacatepay-python-sdk/pull/2/commits/65614f08b74f54ebf244359205d2d01527cd4d62